blocks/library/heading/index.js

```diff
@@ -29,6 +29,8 @@ registerBlockType( 'core/heading', {
 
 	className: false,
 
+	supportAnchor: true,
```

Comment:
Trying to think of another way we might scale this. A similar pattern exists in core with [post type supports](https://developer.wordpress.org/reference/functions/add_post_type_support/) and [theme supports](https://developer.wordpress.org/reference/functions/add_theme_support/). It's a bit different in that case because those supports are always opt-in, which could be represented as an array. If we need some default-true supports, maybe it could still be defined as an object?

```js
supports: {
	className: false,
	anchor: true,
},
```

And we could then have a utility method for getting block supports accounting for the default.

Are there other "supports" types we could see adding in the future?

Should `className` be opt-in? Then we could have:

```js
supports: [
	'className',
	'anchor',
],
```